devLib/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# CMakeLists.txt for devLib
+
+cmake_minimum_required(VERSION 3.10)
+project(devLib)
+
+# Source files
+set(SRC
+    ds1302.c maxdetect.c piNes.c
```

Comment:
May use `file(GLOB)` to build the list: https://cmake.org/cmake/help/v3.30/command/file.html#glob.